src/uberjob/_execution/run_physical.py

```diff
@@ -55,7 +55,7 @@ def _create_bound_call_lookup_and_output_slot(
     plan: Plan, output_node: Optional[Node] = None
 ):
     result_lookup = {
-        node: Slot(node.value if type(node) is Literal else None)
+        node: node if type(node) is Literal else Slot(None)
```

Comment:
This is a tiny step towards eliminating Slot from run_physical entirely, which is possible but I don't want to do as part of this PR.